dashboard/config/routes.rb

```diff
@@ -339,7 +339,7 @@
     # /s/xxx/reset
     get 'reset', to: 'script_levels#reset'
     get 'next', to: 'script_levels#next'
-    get 'hidden_stages', to: 'script_levels#hidden_lesson_ids'
+    get 'hidden_lessons', to: 'script_levels#hidden_lesson_ids'
```

Comment:
this isn't totally safe -- people who have the page open will get 404s when the hidden_stages route goes away. the safe way would be to keep the old route and add the new route. then take the old route away after 1-2 deploys.